### ISSUE TITLE
lsp-process interfere telescope's which-key aciton

### ISSUE THREAD

**Issue by aceforeverd** [linrongbin16/lsp-progress.nvim#50]
steps:

- :Telescope 
- `<c-/>`

Generally, the top-most which-key will exists util any action applyed, with lsp-process, it will be closed automatically. 

example config
```vim
set nocompatible

lua << EOF
local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({
    "git",
    "clone",
    "https://github.com/folke/lazy.nvim.git",
    lazypath,
  })
end

vim.opt.rtp:prepend(lazypath)

require('lazy').setup({
  {
    'nvim-telescope/telescope.nvim',
    dependencies = {
      'nvim-lua/plenary.nvim',
    },
    config = function()
      require('telescope').setup({})
    end
  },

  {
    'linrongbin16/lsp-progress.nvim',
    config = function()
      require('lsp-progress').setup()
    end,
  },
},
{})
EOF
```

**Comment by linrongbin16** [comment]
Thanks, will check

**Comment by linrongbin16** [comment]
Hi @aceforeverd , this issue should be fixed by #51 , please check.

**Comment by aceforeverd** [comment]
> Hi @aceforeverd , this issue should be fixed by #51 , please check.

Fixed for me. Thank you!

**Comment by linrongbin16** [comment]
> > Hi @aceforeverd , this issue should be fixed by #51 , please check.
> 
> Fixed for me. Thank you!

thank you for high quality issue report! bug locating is quite fast for me.
